Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
+
+WORKDIR /opt/app
+
+COPY . /opt/app
```

Comment:
Please consider copying files first and then making the destination as the working directory.